ReactAndroid/src/main/java/com/facebook/react/uimanager/events/Event.java

```diff
@@ -30,15 +30,15 @@
   protected Event() {
   }
 
-  protected Event(int viewTag) {
+  protected Event(Object viewTag) {
     init(viewTag);
   }
 
   /**
    * This method needs to be called before event is sent to event dispatcher.
    */
-  protected void init(int viewTag) {
-    mViewTag = viewTag;
+  protected void init(Object viewTag) {
+    mViewTag = (int) viewTag;
```

Comment:
I'm essentially swapping `setId` and `setTag`.  Instead of using `setId` for the react tags I'm using `setTag`.